src/test/java/org/javarosa/xpath/test/XPathEvalTest.java

```diff
@@ -58,8 +58,8 @@ private void testEval(String expr, FormInstance model, EvaluationContext ec, Obj
             if (exceptionExpected) {
                 fail("Expected exception, expression : " + expr);
             } else if ((result instanceof Double && expected instanceof Double)) {
-                Double o = (Double)result;
-                Double t = (Double)expected;
+                Double o = (Double) result;
```

Comment:
Yeah. All the formatting changes are due to Android Studio auto format. I think we really need to have a standard style.xml that we can import in Android Studio and which Android Studio can then utilise to format/style the code to avoid these conflicts. Though for now will revert these changes.